common/PrerenderData.jsm

```diff
@@ -63,7 +63,8 @@ this.PrerenderData = new _PrerenderData({
     "feeds.section.topstories": true,
     "feeds.section.highlights": true,
     "sectionOrder": "topsites,topstories,highlights",
-    "collapsed": false
+    "collapsed": false,
+    "fxaccounts.endpoint": "https://accounts.firefox.com"
```

Comment:
So this won't be the right approach either. Remove this from here, remove the pref from ActivityStream.jsm, and add a look up for the pref: identity.fxaccounts.remote.root with a fallback value in PrefsFeed after values.platform:

```js
values.fxa_endpoint = Services.prefs.getStringPref("identity.fxaccounts.remote.root", "https://accounts.firefox.com");
```

Then in StartupOverlay.jsx connect Prefs:
```js
const getState = state => ({fxa_endpoint: state.Prefs.values.fxa_endpoint});
export const StartupOverlay = connect(getState)(injectIntl(_StartupOverlay));
```

And then you can use this.props.fxa_endpoint directly in the component

Reply:
Amazing!